CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+New Contributors are always welcome!
+<br>
+Althought most Contributors are [members](https://www.openhwgroup.org/membership/) of the
+OpenHW Group and participate in one or more [Technical Task Groups](https://www.openhwgroup.org/working-groups/),
+membership is not required to be an active Contributor.
+
+## Contributor Agreement
+All Contributors must be covered by the terms of the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
+(for individuals) **or** the [Eclipse Member Committer and Contributor Agreement](https://www.eclipse.org/legal/committer_process/EclipseMemberCommitterAgreement.pdf)
+(for employees of Member companies). The ECA/MCCA provides a legal
+framework for a Contributor's technical contributions to the OpenHW Group,
+including provisions for grant of copyright license and a Developer
+Certificate of Origin on contributions merged into OpenHW Group repositories.
+
+## The Mechanics
```

Comment:
Might be worth to mention `core-v-verif`, and the `cv32e40x/dev` branch, and the `bin/clonetb` script (as described in the README of the `core-v-verif/cv32e40x/` directory).

Reply:
That is a good idea. I'm not 100% clear on the wording for that yet, so I'd like to merge in this legaleze now.  I'll create an issue to get the above added in later.